### PR TITLE
Farson map fixes

### PR DESCRIPTION
**Pull request by AMcilraith**
COOL NEW SUPERMATTER SHIDD

### CODE REVIEW

File: maps/torch/torch6_bridge.dmm [hunk 1]
```diff
@@ -17982,9 +17994,19 @@
...
+	anchored = 1;
+	desc = "It's a card-locked storage unit. ONLY FOR USE IN EXTREME EMERGENCY.";
+	name = "DELTA LEVEL SAFE";
+	pixel_x = -25;
+	req_access = list("ACCESS_VAULT");
+	wall_mounted = 1
+	},
+/obj/item/weapon/disk/nuclear,
+/obj/item/weapon/pinpointer/advpinpointer,
+/obj/item/weapon/folder/envelope/nuke_instructions,
```
Comment (Codingale) [inline maps/torch/torch6_bridge.dmm:18009]: Just checking were does the Nuke disk spawn before this? Like on a table or something?

File: modular_boh/code/modules/torch_security_state.dm [hunk 2]
```diff
@@ -111,15 +111,15 @@
...
+	up_description = "A severe emergency has occurred. Now sounding General Quarters. All essential ship personnel are restricted to their departments, to await further orders. Security may search any individual with due cause, and may have weapons visible on their person and unholstered. All combat ready personnel are to report to security for orders. Lethals are now permitted, and nonessential crew must report to the nearest saferoom, as being visible in general areas is strictly prohibited."
 	psionic_control_level = PSI_IMPLANT_DISABLED
 
 	var/static/datum/announcement/priority/security/security_announcement_red = new(do_log = 0, do_newscast = 1, new_sound = sound('sound/misc/redalert1.ogg'))
 
 /decl/security_level/default/torchdept/code_red/switching_up_to()
 	security_announcement_red.Announce(up_description, "Attention! Code red alert procedures now in effect!")
 	notify_station()
-	GLOB.using_map.unbolt_saferooms()
+	GLOB.using_map.unbolt_saferooms()
```
Comment (Codingale) [inline modular_boh/code/modules/torch_security_state.dm:122]: extra whitespace

File: code/_helpers/matrices.dm [hunk 3]
```diff
@@ -118,4 +118,22 @@
...
 #undef LUMR
 #undef LUMG
-#undef LUMB
\ No newline at end of file
+#undef LUMB
+
+//Matrix math
+//Given 2 matrices mxn and nxp (row major) it multiplies their members and return an mxp matrix
+//Do make sure your lists actually have this many elements
+/proc/multiply_matrices(list/A, list/B, m, n, p)
```
Comment (Codingale) [inline code/_helpers/matrices.dm:126]: Is this used elsewhere?

Comment (AMcilraith) [reply code/_helpers/matrices.dm:126]: not that I know of

File: maps/torch/torch6_bridge.dmm (hunk 1 above)
Comment (AMcilraith) [reply maps/torch/torch6_bridge.dmm:18009]: No, in a closed locker

File: code/_helpers/matrices.dm (hunk 3 above)
Comment (Codingale) [reply code/_helpers/matrices.dm:126]: check for references, where'd you even get this lol

File: maps/torch/torch3_deck3.dmm [hunk 4]
```diff
@@ -8289,7 +8305,6 @@
 "rL" = (
 /obj/machinery/door/firedoor,
 /obj/effect/floor_decal/industrial/hatch/yellow,
-/obj/effect/floor_decal/spline/fancy/wood,
```
Comment (Codingale) [inline maps/torch/torch3_deck3.dmm:8292]: what is this?

**Comment by Codingale** [comment]
Farson said MS office signed off on it, so I'll merge this.

**Pull request merged**